Forge.TreeWalker.UnitTests/test/TreeWalkerUnitTests.cs

```diff
@@ -67,19 +100,49 @@ public void TestInitialize(string jsonSchema, string treeName = null)
             this.session = new TreeWalkerSession(this.parameters);
         }
 
+        /// <summary>
+        /// Helper function to test V1 ITreeWalkerCallbacks on the contructor accepting ForgeTree.
+        /// </summary>
+        /// <param name="forgeTree">The ForgeTree object.</param>
+        /// <param name="treeName">The tree name.</param>
+        public void TestInitializeWithForgeTree_WithV1Callbacks(ForgeTree forgeTree, string treeName = null)
+        {
+            // Initialize contexts, callbacks, and actions.
+            this.sessionId = Guid.NewGuid();
+            this.forgeState = new ForgeDictionary(new Dictionary<string, object>(), this.sessionId, this.sessionId);
+            TreeWalkerCallbacks treeWalkerCallbacksV1 = new TreeWalkerCallbacks();
+            this.token = new CancellationTokenSource().Token;
+
+            this.parameters = new TreeWalkerParameters(
+                this.sessionId,
+                forgeTree,
+                this.forgeState,
+                treeWalkerCallbacksV1,
+                this.token)
+            {
+                UserContext = new ForgeUserContext(),
+                ForgeActionsAssembly = typeof(CollectDiagnosticsAction).Assembly,
+                InitializeSubroutineTree = this.InitializeSubroutineTree,
+                TreeName = treeName,
+                Dependencies = new List<Type>() { typeof(FooEnum) }
+            };
+
+            this.session = new TreeWalkerSession(this.parameters);
+        }
+
         public void TestInitializeWithForgeTree(ForgeTree forgeTree, string treeName = null)
```

Comment:
>public void TestInitializeWithForgeTree [](start = 8, length = 39)

Let's move this initializer just under TestInitialize(..) so they are next to each other. The V1 initializers can follow.

Reply:
moved the V1 initializers to follow the regular initializers.

---
In reply to: [506625855](https://github.com/microsoft/Forge/pull/32#discussion_r506625855) [](ancestors = 506625855)